menu.js

```diff
@@ -31,8 +142,208 @@ document
   .getElementsByClassName('portfolio')[0]
   .addEventListener('click', closeMenu);
 document
-  .getElementsByClassName('about')[0]
+  .getElementsByClassName('About')[0]
   .addEventListener('click', closeMenu);
 document
   .getElementsByClassName('contact')[0]
-  .addEventListener('click', closeMenu);
\ No newline at end of file
+  .addEventListener('click', closeMenu);
+document.addEventListener('keydown', xButton);
+
+document.getElementById('See1').addEventListener('click', openPopup1);
+document
+  .getElementsByClassName('cancel-1')[0]
+  .addEventListener('click', () => {
+    closePopup1();
+  });
+
+document.getElementById('See2').addEventListener('click', openPopup2);
+document.getElementById('SeeD2').addEventListener('click', openPopup2);
+document
+  .getElementsByClassName('cancel-2')[0]
+  .addEventListener('click', closePopup2);
+
+document.getElementById('See3').addEventListener('click', openPopup3);
+document.getElementById('SeeD3').addEventListener('click', openPopup3);
+document
+  .getElementsByClassName('cancel-3')[0]
+  .addEventListener('click', closePopup3);
+
+document.getElementById('See4').addEventListener('click', openPopup4);
+document.getElementById('SeeD4').addEventListener('click', openPopup4);
+document
+  .getElementsByClassName('cancel-4')[0]
+  .addEventListener('click', closePopup4);
+
+document.getElementById('See5').addEventListener('click', openPopup5);
+document.getElementById('SeeD5').addEventListener('click', openPopup5);
+document
+  .getElementsByClassName('cancel-5')[0]
+  .addEventListener('click', closePopup5);
+
+document.getElementById('See6').addEventListener('click', openPopup6);
+document.getElementById('SeeD6').addEventListener('click', openPopup6);
+document
+  .getElementsByClassName('cancel-6')[0]
+  .addEventListener('click', closePopup6);
+
+const storeObj = {
+  1: {
+    name: 'Project name goes here',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat.  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.',
+    featuredImg: [
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
+      'image/Popup-menu/popup-1/Rectangle21.jpg',
```

Comment:
- [ ] kindly centralize your bottom text to follow the figma design. great job 
yours 
![Screenshot (529)_LI](https://user-images.githubusercontent.com/65298048/195557687-4a619ad2-2fad-4793-8817-2ad6d2a28b07.jpg)
figma
![Screenshot (527)_LI](https://user-images.githubusercontent.com/65298048/195557957-f16e81df-b679-4318-a6b2-94a734584d55.jpg)